package.json

```diff
@@ -33,6 +33,7 @@
     "@syncfusion/ej2-react-calendars": "^20.1.58",
     "aws-sdk": "^2.1098.0",
     "bootstrap": "^5.2.1",
+    "cookies-next": "^2.1.1",
```

Comment:
Kindly bump up @klaudsol/commons version to:

"@klaudsol/commons": "https://github.com/klaudsol/klaudsol-commons.git#v1.1.0"